localstack/testing/pytest/fixtures.py

```diff
@@ -33,6 +33,7 @@
 from localstack.utils.generic.wait_utils import wait_until
 from localstack.utils.net import wait_for_port_open
 from localstack.utils.testutil import start_http_server
+from tests.integration.apigateway_fixtures import create_rest_api, delete_rest_api, import_rest_api
```

Comment:
Let's try to avoid importing code from `tests` within the main `localstack` codebase. I guess we could move these functions from `apigateway_fixtures.py` into this file, and also turn them into real pytest fixtures. Thoughts?

Reply:
yea, absolutely! That was an innocent mistake.